src/PlusOpenIGTLink/PlusIgtlClientInfo.cxx

```diff
@@ -53,12 +54,19 @@ PlusStatus PlusIgtlClientInfo::SetClientInfoFromXmlData(vtkXMLDataElement* xmlda
 
   if (xmldata->GetAttribute("TDATARequested") != NULL)
   {
-    clientInfo.TDATARequested = STRCASECMP(xmldata->GetAttribute("TDATARequested"), "TRUE") == 0;
+    clientInfo.SetTDATARequested(STRCASECMP(xmldata->GetAttribute("TDATARequested"), "TRUE") == 0);
   }
 
   if (xmldata->GetAttribute("Resolution") != NULL)

```

Comment:
"Resolution" is a really bad parameter name and it is not even documented. Could you please rename it and add documentation (see description here: https://github.com/openigtlink/OpenIGTLink/blob/master/Documents/Protocol/trackingdata.md). For backward compatibility, allow using the old attribute name as well, but log a warning.

Reply:
Agreed, fyi resolution is the name of the parameter for the TDATA message.